packages/ui-kit/src/hooks/useTopValues/useTopValues.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, RelativeTimeRange, useStorybookAccessToken } from '../../helpers'
+
+import { useTopValues } from './useTopValues'
+import { TopValuesQueryProps } from '../../components/TopValues/TopValues.types'
+
+const args: TopValuesQueryProps = {
+  dataPool: {
+    name: 'TacoSoft Demo Data'
```

Comment:
Could you please use an env variable for that? Like [here](https://github.com/propeldata/ui-kit/blob/08f9f4d867734c983da44245fb9c10ecdada24af/packages/ui-kit/src/hooks/useLeaderboard/useLeaderboard.stories.tsx#L11).
It will be easier to run Storybook locally.